Makefile

```diff
@@ -9,7 +9,11 @@ build:
 	@yarn buildProd
 
 # Lint code, commits and documentation.
-lint-targets := lint-git
+lint-targets := lint-spell lint-git
+
+lint-spell:
+	@$(ECHO) "$(CYAN)*** Runnint cspell...$(OFF)"
```

Comment:
Runnint :scream:

Reply:
:laughing: Thanks for fixing this in https://github.com/oasisprotocol/oasis-wallet-ext/pull/134/commits/b9d8c499e57c0cd93af35367f0942e9f029d6833.